src/test/java/org/jenkinsci/plugins/argusnotifier/JenkinsRunFormatterTest.groovy

```diff
@@ -70,6 +71,43 @@ class JenkinsRunFormatterTest extends Specification {
         actualBuildNumberString == String.valueOf(buildNumber)
     }
 
+    def "test getGitCommit returns commit sha"() {
+        given:
+        def gitCommit = commitId
+
+        ImmutableMap.Builder<String,String> mapBuilder = ImmutableMap.<String, String>builder()
+        mapBuilder.put(JenkinsRunFormatter.GIT_COMMIT, gitCommit)
+        build.getEnvVars() >> mapBuilder.build()
+        JenkinsRunFormatter jenkinsBuildFormatter = new JenkinsRunFormatter(jenkins, build)
+
+        when:
+        String actualGitCommit = jenkinsBuildFormatter.getGitCommit()
+
+        then:
+        actualGitCommit == expectedGitCommit
+
+        where:
+        commitId                                   | expectedGitCommit
+        "0fc255bb26fcc4a0548c3ca14caec6a7d6de7c25" | "0fc255bb26fcc4a0548c3ca14caec6a7d6de7c25"
+        "f6fd56be96ccb7f445c6d8058b76b4c2482a50a6" | "f6fd56be96ccb7f445c6d8058b76b4c2482a50a6"
+    }
+
+    def "test getGitCommit no such env variable"() {
+        given:
+        def gitCommit = "343245dsawer32fd43fd43"
+
+        ImmutableMap.Builder<String,String> mapBuilder = ImmutableMap.<String, String>builder()
+        mapBuilder.put("ENV_VAR", gitCommit)
+        build.getEnvVars() >> mapBuilder.build()
```

Comment:
You can squash the above builder lines as well as `gitCommit` (not sure that it adds any value).
```suggestion
        build.getEnvVars() >> [:]
```